src/Adapter/LegacyContext.php

```diff
@@ -214,6 +214,8 @@ public function getLegacyLayout(
         $metaTitle = '',
         $useRegularH1Structure = true
     ) {
+        $previousContextController = $this->getContext()->controller;
```

Comment:
You maybe need to check there is a controller in the context? 
I can't trust the `getContext` method :sweat_smile:

Reply:
Trust me 🤩

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19277/commits/a4cd78e376c097f7b5c407e4934d715432c7e2b6